scripts/dts/extract/globals.py

```diff
@@ -546,10 +552,13 @@ def extract_cells(node_path, prop, prop_values, names, index,
                     node_path,
                     lambda alias: '_'.join([str_to_label(alias)] + label[1:]),
                     '_'.join(label),
-                    prop_alias)
+                    prop_alias, deprecate)
 
             insert_defs(node_path, prop_def, prop_alias)
 
+            if deprecate:
+                deprecated_main.extend(list(prop_def.keys()))
```

Comment:
`deprecated_main.extend(prop_def.keys())` (or `deprecated_main.extend(prop_def)`) will work too btw. Python allows "iterable" things to be used where lists are.